pom.xml

```diff
@@ -66,6 +66,11 @@ limitations under the License.
     </modules>
 
     <dependencies>
+        <dependency>
+            <groupId>org.lz4</groupId>
```

Comment:
It seems like we're using this dependency only in the runtime executor. It might be better to move this into the appropriate pom.xml

Reply:
@wonook I fixed it. Thank you!